### PR TITLE
Sign up and login thumbnail issue resolved

### LINKED ISSUE #254: SignUp and LogIn page icon issue
Two Pages:-
https://circuitverse.org/users/sign_up
https://circuitverse.org/users/sign_in

gsoc_LI
gsoc is_LI

In LogIn page
1--"Sign up with Google" will be "Continue with Google"
2-- There will be a facebook icon attached with "Continue with Facebook"

In SignUp page
1--"Log in with Facebook" will be "Continue with Facebook"
2-- And also a facebook icon 

Instead of writing those things i have some suggestions.Check the Following pic.

Screenshot (10)

Are you working on this issue? 
Yes

### PR DESCRIPTION
**Pull request by tabishshakeel**
Fixes #254 (Added by @armudgal)

On the both pages of signup and login it was showing the same thumbnails.
login page
signup page

### CODE REVIEW

**Comment by tabishshakeel** [comment]
@armudgal here is my pull request. Please review it once.

**Comment by armudgal** [comment]
IDK which issue it refers to, please follow the PR template

**Comment by tachyons** [comment]
Also can make sure that buttons are aligned properly ? Buttons in screenshot looks ugly to me 

Some tips here https://uxplanet.org/designing-ux-login-form-and-process-8b17167ed5b9

**Comment by tabishshakeel** [comment]
@armudgal  the buttons are looking good without outlining so I think we should not add outlines on the buttons.
Screenshot from 2019-03-22 17-39-00

**Comment by tabishshakeel** [comment]
> @armudgal the buttons are looking good without outlining so I think we should not add outlines on the buttons.
> Screenshot from 2019-03-22 17-39-00

please let me know about any other changes if not then I'll submit another PR.

File: Gemfile [hunk 1]
```diff
@@ -102,5 +95,10 @@ group :development do
   gem 'rails-erd'
 end
 
+group :production do
+  gem 'pg', '1.1.4'
+  gem 'rails_12factor'
+end
+
```
Comment (satu0king) [inline Gemfile]: Why are you having changes in the Gemfile?

**Comment by satu0king** [comment]
Please resolve the conflicts

File: Gemfile (hunk 1 above)
Comment (satu0king) [reply Gemfile]: We are using postgres in development as well now.

**Comment by satu0king** [comment]
I think the current sign up/ sign in page is working fine.

**Pull request closed without merging**